src/styles/header.scss

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), styles for `html` should not be set in component files like `header.scss`. These should be defined in a base stylesheet such as `reset.scss` or `index.scss`.